bot.py

```diff
@@ -335,86 +335,88 @@ async def verify(ctx):
+            ('🇨🇿', "cz", "Czechia", ["Czech"]),
+            ('🇩🇰', "dk", "Denmark", ["Danish"]),
+            ('🇪🇪', "ee", "Estonia", ["Estonian"]),
+            ('🇫🇮', "fi", "Finland", ["Finnish", "Swedish"]),
+            ('🇫🇷', "fr", "France", ["French"]),
+            ('🇩🇪', "de", "Germany", ["German"]),
+            ('🇬🇪', "ge", "Georgia", ["Georgian"]),
+            ('🇬🇷', "gr", "Greece", ["Greek"]),
+            ('🇭🇺', "hu", "Hungary", ["Hungarian"]),
+            ('🇮🇸', "is", "Iceland", ["Icelandic"]),
+            ('🇮🇪', "ie", "Ireland", ["Irish"]),
+            ('🇮🇹', "it", "Italy", ["Italian"]),
+            ('🇽🇰', "xk", "Kosovo", ["Kosovar"]),
+            ('🇰🇿', "kz", "Kazakhstan", ["Kazakh"]),
+            ('🇱🇻', "lv", "Latvia", ["Latvian"]),
+            ('🇱🇮', "li", "Liechteinstein", ["German"]),
+            ('🇱🇹', "lt", "Lithuania", ["Lithuanian"]),
+            ('🇱🇺', "lu", "Luxembourg", ["Luxembourgish", "French", "German"]),
+            ('🇲🇹', "mt", "Malta", ["Maltese"]),
+            ('🇲🇩', "md", "Moldova", ["Romanian"]),
+            ('🇲🇨', "mc", "Monaco", ["French"]),
+            ('🇲🇪', "me", "Montenegro", ["Montenegrin"]),
+            ('🇳🇱', "nl", "Netherlands", ["Dutch"]),
+            ('🇲🇰', "mk", "North Macedonia", ["Macedonian", "Albanian"]),
+            ('🇳🇴', "no", "Norway", ["Norwegian"]),
+            ('🇵🇱', "pl", "Poland", ["Polish"]),
+            ('🇵🇹', "pt", "Portugal", ["Portuguese"]),
+            ('🇷🇴', "ro", "Romania", ["Romanian"]),
+            ('🇷🇺', "ru", "Russia", ["Russian"]),
+            ('🇸🇲', "sm", "San Marino", ["Italian"]),
+            ('🇷🇸', "rs", "Serbia", ["Serbian"]),
+            ('🇸🇰', "sk", "Slovakia", ["Slovak"]),
+            ('🇸🇮', "si", "Slovenia", ["Slovenian"]),
+            ('🇪🇸', "es", "Spain", ["Spanish"]),
+            ('🇸🇪', "se", "Sweden", ["Swedish"]),
+            ('🇨🇭', "ch", "Switzerland", ["German", "French", "Italian"]),
+            ('🇹🇷', "tr", "Turkey", ["Turkish"]),
+            ('🇬🇧', "gb", "United Kingdom", []),
+            ('🇺🇦', "ua", "Ukraine", ["Ukrainian", "Russian"]),
+            ('🇻🇦', "va", "Vatican", ["Italian"]),
+            (':region_asia:', "asia", "Asia", []),
+            (':region_africa:', "africa", "Africa", []),
+            (':region_northamerica:', "us", "North America", []),
+            (':region_northamerica:', "ca", "North America", []),
+            (':region_oceania:', "oceania", "Oceania", []),
+            (':region_southamerica:', "south_america", "South America", [])
         ]
 
-        countries = list(emojis.get(ctx.message.content))
+        msgContent = ctx.message.content
+        countries = list(emojis.get(msgContent))
         reg = regex.compile(r"<(:\w+:)\d+>")
-        countries += reg.findall(ctx.message.content)
+        countries += reg.findall(msgContent)
+
+        country_codes = set(x.lower() for x in msgContent[len(constantes.prefixVolt)+len("verify "):].split("+")[0].split("-")[0].split())
```

Comment:
Just split by space. Is there ever a collision of a country code and a language?

Reply:
No. Now I think there is a more convenient way of handling this, I'm making a commit soon.